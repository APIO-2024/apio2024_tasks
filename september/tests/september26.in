txy4h26c1rm1uv8tr3eonahd67u8h56x
53
10 1
0 0 0 1 1 1 4 2 3
5 7 9 3 8 4 2 1 6
10 1
0 0 1 1 1 1 3 2 6
7 1 4 9 6 3 2 5 8
10 1
0 0 2 2 2 2 3 0 7
1 6 4 9 8 7 3 5 2
10 1
0 0 2 1 3 5 6 6 1
4 6 8 7 9 5 3 1 2
10 1
0 0 1 3 3 1 1 2 1
6 5 4 7 8 9 3 1 2
10 1
0 1 0 1 4 5 3 2 7
9 8 2 7 6 4 5 3 1
10 1
0 1 1 1 3 4 3 0 6
5 7 8 2 9 3 6 4 1
10 1
0 0 0 2 0 4 1 6 5
9 3 5 8 7 2 4 1 6
10 1
0 0 2 1 0 1 1 0 7
3 8 6 4 2 9 5 7 1
10 1
0 1 1 2 3 0 5 6 1
7 8 9 4 3 5 6 2 1
10 1
0 0 2 3 4 1 0 3 8
5 4 9 6 8 1 2 3 7
10 1
0 1 2 0 2 2 0 5 8
7 3 9 6 4 8 5 2 1
10 1
0 0 1 1 1 1 2 1 1
7 4 6 3 8 5 2 9 1
10 1
0 1 0 3 1 3 3 2 4
8 9 6 4 5 2 1 7 3
10 1
0 0 0 2 4 2 1 7 7
6 7 1 8 2 9 4 5 3
10 1
0 1 2 2 1 3 0 5 5
9 4 6 7 2 3 1 5 8
10 1
0 1 2 3 4 5 2 5 3
8 7 6 5 9 4 3 2 1
10 1
0 1 2 0 0 5 3 4 0
9 7 6 8 3 5 4 2 1
10 1
0 0 1 2 2 4 1 0 2
3 8 6 4 9 5 7 2 1
10 1
0 1 0 2 2 5 1 7 0
3 4 2 5 6 8 9 7 1
40 1
0 0 1 0 2 1 3 0 3 5 7 6 2 5 1 8 1 7 9 16 6 13 3 10 23 16 20 14 23 14 21 10 8 2 21 2 12 10 13
15 36 22 30 25 32 39 28 29 24 27 18 19 37 9 38 10 26 11 20 4 12 13 31 35 33 23 17 21 6 14 7 16 8 3 5 34 2 1
40 1
0 1 2 2 2 3 6 7 5 7 1 0 10 1 4 9 4 10 4 6 3 16 1 1 1 19 24 18 20 18 19 29 1 9 16 16 2 33 15
25 14 23 36 17 22 11 13 26 32 28 33 35 31 21 39 38 16 19 27 8 30 37 24 15 12 34 29 9 20 6 5 10 7 18 1 3 2 4
40 1
0 1 1 0 4 4 3 1 3 0 0 0 7 5 7 3 0 10 4 13 9 9 19 21 21 7 4 7 19 21 2 27 1 13 32 20 17 17 4
25 6 12 14 8 39 16 15 29 11 38 22 31 37 5 35 27 32 17 28 33 36 2 30 23 19 26 20 18 4 24 34 10 13 9 7 21 3 1
40 1
0 1 1 3 4 1 4 6 8 0 4 5 6 8 0 1 13 2 2 16 4 18 11 2 0 4 10 9 22 20 8 10 6 15 21 23 9 36 16
28 17 13 39 24 30 7 32 25 20 31 19 38 10 27 14 12 26 35 9 37 34 5 29 8 22 18 36 16 15 33 2 23 6 21 11 3 4 1
40 1
0 1 0 3 3 4 0 4 1 5 5 1 10 6 10 15 8 6 14 6 5 20 10 22 21 22 17 13 14 17 12 2 30 8 28 32 35 32 15
39 19 29 27 25 24 37 11 16 14 15 23 36 31 26 38 32 33 12 22 7 2 34 35 9 20 18 21 28 6 30 1 17 10 13 5 4 3 8
100 1
0 1 1 2 1 1 2 7 6 9 7 3 2 9 11 2 2 5 11 14 5 2 8 10 17 18 8 6 19 29 1 31 28 3 0 34 31 1 0 29 24 21 23 34 13 16 26 42 27 26 11 24 8 51 3 24 16 46 56 38 31 61 44 0 14 10 50 31 53 48 58 12 57 18 74 2 51 52 4 0 35 4 7 16 13 19 62 0 66 25 7 62 45 28 72 92 50 55 62
49 77 69 40 99 75 96 87 95 90 36 43 30 59 15 25 47 65 93 71 27 73 33 68 41 54 78 91 51 89 17 45 83 20 53 86 57 97 52 76 81 58 39 35 66 82 64 32 67 29 46 85 79 84 98 80 63 55 19 16 11 37 88 14 4 22 50 92 94 60 28 56 72 44 62 61 26 31 38 13 70 34 23 48 12 74 3 8 42 7 24 2 18 21 5 10 9 6 1
1000 1
0 1 0 2 4 2 1 7 2 3 0 6 6 3 1 14 7 16 1 3 14 8 19 15 23 6 13 14 4 4 22 6 1 27 31 18 23 22 0 26 33 4 10 9 5 17 27 38 23 43 45 48 51 13 19 37 7 15 31 34 7 42 42 26 59 15 30 16 59 22 49 40 48 18 65 32 33 45 5 69 8 29 32 29 59 14 46 27 14 16 64 63 70 28 92 0 5 96 29 86 60 59 65 23 31 80 24 0 91 43 11 66 88 58 89 11 20 83 41 118 29 10 122 113 68 60 45 56 118 50 24 53 17 5 94 84 82 37 24 65 83 11 73 47 6 81 100 59 100 27 9 62 68 150 104 93 141 40 41 46 42 153 21 19 119 91 77 42 34 98 18 1 118 28 109 78 26 47 128 107 161 144 141 146 39 114 31 163 84 174 92 185 101 48 187 140 29 187 151 161 195 82 183 192 138 83 77 144 147 125 41 39 200 57 98 56 4 205 99 74 110 212 158 18 50 90 132 178 35 157 92 178 98 170 97 3 63 152 104 161 28 155 193 136 49 148 60 174 19 76 44 123 48 211 221 198 91 18 187 100 229 152 75 162 122 186 154 146 220 84 27 215 135 214 195 256 257 260 29 192 64 94 154 166 240 58 196 279 221 44 186 228 86 89 4 71 111 28 237 157 268 116 259 2 185 14 297 97 277 172 310 95 23 127 84 103 231 245 267 77 203 284 185 253 189 162 240 125 251 115 90 45 78 260 72 64 196 199 221 147 103 146 60 306 317 290 83 201 27 134 319 275 162 134 232 130 240 15 302 253 93 195 312 242 35 362 39 149 300 130 106 119 40 57 12 300 150 168 211 262 361 125 375 217 336 195 161 275 199 217 331 90 306 56 12 188 52 134 345 276 59 10 20 289 83 45 219 395 220 294 62 134 295 298 115 54 5 387 313 41 189 74 169 115 16 141 119 40 301 156 423 376 342 92 403 346 328 114 219 276 358 142 172 405 396 327 241 377 228 446 286 98 405 169 308 381 186 420 1 428 331 228 261 394 440 287 152 38 304 22 189 330 355 75 62 37 235 56 472 402 72 221 473 120 144 417 97 375 460 298 399 62 61 58 208 401 482 83 413 371 394 372 397 449 431 85 256 30 43 47 235 417 300 213 310 408 444 83 316 44 205 420 141 111 517 376 362 242 288 513 101 308 257 134 324 261 398 211 60 298 134 348 341 160 504 450 32 273 210 445 406 429 170 229 547 71 556 544 346 470 149 136 98 217 89 71 105 157 410 472 86 490 109 444 157 150 460 503 235 336 424 133 381 281 567 514 209 165 105 72 407 325 140 592 463 100 466 558 161 333 86 503 474 432 395 211 208 465 243 143 560 18 428 453 520 351 474 541 143 167 109 295 513 224 168 505 80 223 477 559 535 220 530 203 259 136 494 28 194 595 575 624 540 577 629 293 392 21 493 170 157 215 242 70 299 286 43 432 640 15 508 567 525 104 370 658 70 400 52 97 246 155 583 616 165 514 11 377 75 527 171 195 434 443 578 36 53 1 284 45 319 72 446 542 339 2 203 633 540 262 631 578 581 33 16 420 402 457 93 686 435 218 631 593 383 70 638 51 497 572 118 233 274 696 552 1 720 141 212 718 49 375 403 545 344 631 353 631 479 623 161 408 101 420 612 397 501 710 627 43 710 497 459 150 438 316 167 294 553 372 540 383 114 353 471 444 293 266 393 82 354 486 394 161 595 479 481 418 5 325 760 413 455 668 404 606 214 517 471 241 211 166 286 312 134 653 623 418 624 106 118 601 127 239 425 410 313 286 267 127 413 784 349 284 248 701 553 4 622 779 529 282 167 652 791 6 638 84 575 490 294 483 143 794 32 768 425 178 131 100 826 126 602 76 687 517 404 803 723 333 672 65 708 446 142 232 113 331 660 13 340 572 428 202 435 832 416 661 147 335 163 736 734 611 133 87 283 525 877 494 666 484 140 823 180 128 197 90 45 669 124 141 606 691 184 345 300 487 83 514 575 546 521 679 194 826 716 798 67 231 205 771 214 842 360 891 805 672 665 729 753 333 760 69 358 847 241 214 288 95 306 580 859 150 440 111 218 923 766 178 283 734 814 96 587 708 642 622 861 44 733 421 554 741 779 551 158 283 727 288 687 890 920 70 464 741 752 609 719 51 789 453 243 307 930 378 947 190 857 944 139 331 773 200 814 642 452 179 613 340 763 212 165 49 853 363 729 676 350
735 846 997 929 309 933 905 761 780 545 965 534 586 976 562 679 885 830 541 239 743 620 690 561 860 894 765 945 314 920 587 939 871 177 343 129 738 854 797 969 531 625 902 234 550 813 139 176 565 983 647 618 566 853 621 456 811 726 831 734 749 775 393 250 657 822 265 568 912 781 682 936 398 968 499 108 778 995 173 492 102 363 745 296 603 680 454 698 491 998 754 711 495 925 121 579 841 670 352 543 685 958 882 573 951 512 641 996 384 574 904 368 451 984 942 748 697 848 307 632 937 159 599 379 206 399 247 548 182 859 112 974 634 285 855 769 694 502 576 369 527 867 915 802 510 833 798 329 533 927 795 804 959 899 511 838 598 411 964 462 270 981 320 924 880 683 897 601 926 537 323 851 820 597 254 278 532 713 689 873 589 117 886 629 782 796 570 590 650 991 908 770 493 488 409 662 812 538 834 709 950 934 849 935 750 645 674 836 869 380 427 970 963 413 549 322 783 649 910 236 441 207 721 25 878 515 414 608 702 489 826 416 952 994 918 733 856 845 342 938 344 305 901 829 893 941 875 277 582 982 883 442 928 128 373 973 772 962 585 408 852 799 931 909 695 366 461 847 132 627 794 433 989 957 426 588 478 547 693 522 944 194 351 506 706 715 906 282 518 269 766 872 691 415 863 374 208 626 390 347 656 648 728 988 956 546 302 271 392 922 864 773 790 895 272 707 516 468 180 116 850 736 354 465 577 890 569 357 762 274 216 828 255 654 266 304 837 898 717 752 704 809 643 480 785 485 746 555 858 972 251 149 315 292 467 985 334 367 536 175 801 887 475 244 755 191 440 252 975 792 881 776 67 723 807 422 954 739 719 109 505 71 967 455 563 730 903 742 358 179 202 131 628 888 987 732 992 664 222 484 612 359 610 663 870 892 644 389 469 718 439 263 419 659 681 771 839 227 607 921 575 280 788 639 437 637 616 768 678 646 55 209 181 729 946 889 986 557 913 857 249 917 509 556 971 651 789 675 388 273 980 85 521 764 979 349 824 653 672 264 911 210 258 519 135 787 508 494 868 914 763 539 215 400 328 164 594 303 212 667 53 606 335 703 481 308 201 808 313 615 341 635 321 584 425 916 421 700 438 391 751 350 211 687 107 731 193 592 948 891 486 818 677 638 613 504 923 466 448 767 816 259 165 947 412 671 879 243 423 640 332 932 287 447 716 406 692 365 814 233 542 803 953 793 431 497 779 907 436 540 464 118 404 642 844 225 401 498 977 701 306 605 865 445 94 144 312 661 145 827 496 430 260 583 54 636 744 832 884 993 652 142 990 684 955 725 666 76 319 961 943 591 668 596 630 331 673 524 609 669 395 61 171 330 220 722 777 800 791 197 581 381 500 297 137 311 529 184 862 784 567 523 747 688 806 238 337 930 458 600 835 551 460 757 686 825 819 223 866 554 578 617 949 410 472 676 810 571 580 79 604 459 526 242 760 774 443 364 158 452 245 353 360 753 479 614 786 192 356 230 418 712 966 737 999 740 471 699 24 377 805 74 710 123 724 655 821 840 434 564 756 450 66 758 874 122 169 120 204 318 474 817 741 133 281 528 348 294 90 453 815 432 310 940 705 34 45 214 103 127 199 64 101 513 178 57 198 560 68 166 63 371 157 324 20 89 355 235 30 619 290 72 111 88 4 96 84 552 407 900 62 167 624 714 611 396 69 514 38 219 146 372 141 473 183 168 476 18 93 470 73 147 114 623 75 156 559 535 362 553 172 152 720 602 375 35 41 417 91 80 383 36 49 46 110 113 284 15 267 21 92 11 299 482 39 10 420 26 333 28 240 435 50 843 402 842 558 340 326 658 217 160 978 382 33 520 224 229 327 174 708 397 507 48 203 213 370 200 65 300 52 126 338 593 226 196 130 325 5 1 256 13 405 286 457 125 43 104 77 241 51 919 153 2 70 136 276 22 7 82 483 59 32 246 378 289 237 339 262 544 633 6 429 660 27 877 17 42 253 727 162 386 823 301 190 316 268 463 154 83 275 477 228 696 87 428 155 23 631 3 134 317 185 151 44 572 105 124 291 503 376 148 9 195 14 759 896 283 95 40 487 138 8 298 186 861 261 221 665 387 622 16 288 248 530 119 336 100 78 163 279 876 394 517 29 424 60 446 218 37 86 187 490 19 403 231 58 449 106 81 143 345 115 98 232 188 361 444 525 205 960 12 31 150 257 293 595 346 295 97 385 189 99 140 56 47 170 161 501
100 1
0 1 2 2 3 0 1 0 3 2 0 7 1 11 5 14 9 6 1 14 10 17 2 12 22 15 14 8 28 14 5 30 31 10 18 11 7 29 33 5 26 25 0 30 22 1 21 17 18 18 32 23 5 15 54 33 19 32 18 26 30 43 14 3 6 55 64 29 23 9 37 13 37 25 17 20 56 33 43 48 40 18 70 79 5 30 17 31 65 0 9 47 3 67 50 30 44 68 44
87 91 34 52 41 59 36 35 49 69 93 90 72 4 71 46 83 77 80 76 42 13 58 51 85 27 98 56 74 94 75 99 60 16 89 25 53 95 24 78 92 61 20 32 73 23 70 50 26 67 40 81 37 86 84 64 88 12 82 62 57 68 48 47 63 39 96 33 21 38 7 19 31 29 45 28 22 65 10 18 17 97 6 66 8 9 44 79 54 55 43 15 30 5 3 14 11 2 1
40 1
0 1 2 2 4 2 5 7 0 9 6 4 12 10 1 11 15 15 3 16 14 3 12 1 1 23 1 10 18 27 24 26 29 32 12 34 19 32 23
31 38 35 36 34 25 8 22 37 21 32 26 30 33 13 29 27 28 17 39 20 14 10 18 19 11 16 3 24 7 23 15 9 5 12 6 4 2 1
40 1
0 1 1 3 1 1 5 0 7 2 6 9 2 10 12 8 12 3 4 8 19 18 9 9 12 1 10 18 20 5 3 28 28 1 8 24 13 37 12
14 32 16 11 6 21 36 23 19 17 26 24 34 29 25 39 33 20 22 30 27 15 31 10 38 37 18 28 13 2 12 35 8 4 7 9 3 1 5
40 1
0 0 1 0 4 5 2 5 2 5 1 10 12 11 12 1 2 5 16 11 1 5 20 6 7 20 25 21 21 29 3 22 1 3 25 30 30 7 4
19 14 8 24 23 15 33 36 6 9 35 31 13 18 38 34 17 39 32 37 22 30 27 26 20 28 3 25 29 16 11 12 7 2 5 21 1 10 4
40 1
0 1 0 2 2 3 1 4 1 1 7 1 11 4 7 2 6 14 7 13 2 15 15 13 2 19 3 26 13 6 22 19 28 26 5 31 7 35 5
18 23 10 25 36 34 33 12 24 9 30 17 6 20 16 27 21 14 28 32 29 8 13 26 3 37 31 19 11 39 22 4 38 5 35 15 1 2 7
40 1
0 0 1 1 0 4 1 2 1 4 3 5 7 0 3 1 11 8 4 12 16 8 11 7 19 17 15 0 21 9 26 5 5 33 20 21 36 1 19
24 27 39 32 37 30 31 9 35 29 25 22 23 18 28 15 13 34 17 10 26 8 6 19 7 14 4 5 36 21 16 3 20 12 1 2 33 11 38
10 1
0 1 1 2 0 3 1 4 8
6 7 9 8 3 4 5 2 1
10 1
0 1 1 0 2 4 0 4 8
5 7 9 2 3 6 1 8 4
10 1
0 1 0 2 1 4 0 6 8
5 7 9 8 3 6 4 1 2
10 1
0 1 2 2 0 5 5 1 4
6 9 8 3 4 7 2 5 1
10 1
0 1 1 3 3 3 6 6 0
4 8 7 5 9 2 3 6 1
10 1
0 1 1 2 0 0 4 4 5
6 8 9 7 5 4 3 2 1
10 1
0 0 2 2 0 1 5 2 8
9 7 3 4 6 5 1 2 8
10 1
0 1 2 3 2 3 5 4 5
8 9 4 5 6 7 3 2 1
10 1
0 0 1 1 3 4 0 3 0
6 9 8 5 2 3 7 4 1
10 1
0 0 2 2 4 4 3 3 8
5 1 7 9 2 3 8 4 6
10 1
0 0 0 1 3 1 5 7 0
9 6 8 2 7 1 4 5 3
10 1
0 1 1 0 1 2 2 3 4
9 6 8 7 4 5 1 3 2
10 1
0 0 0 0 0 3 0 2 5
4 8 6 7 3 1 2 9 5
10 1
0 0 1 3 1 4 5 2 3
6 9 7 3 8 5 4 2 1
10 1
0 0 2 2 1 1 0 3 1
9 7 8 4 1 6 5 3 2
10 1
0 1 2 0 0 1 2 1 4
5 9 4 7 6 3 8 2 1
10 1
0 1 1 0 0 1 0 7 8
2 9 4 8 7 3 1 5 6
10 1
0 0 1 1 1 5 0 0 2
7 8 9 3 2 4 6 5 1
10 1
0 1 0 3 0 4 1 7 8
5 2 6 8 9 7 1 4 3
10 1
0 1 0 1 0 4 3 5 1
2 1 3 7 9 5 4 8 6
