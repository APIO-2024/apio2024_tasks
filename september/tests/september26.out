7ckgnn4wyi495puj3ibqf81dqvapyv6b
8
2
9
7
9
8
9
6
9
8
8
9
9
9
3
6
9
9
9
7
39
27
36
36
36
99
37
97
38
36
36
36
26
9
9
8
9
8
9
8
7
9
5
8
7
9
6
7
9
7
9
8
2
